ballerina-shell/modules/shell-core/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     implementation project(':ballerina-lang')
     implementation project(':ballerina-tools-api')
     implementation project(':ballerina-runtime')
+    implementation project(':identifier-util')
```

Comment:
Since Utils is a more common class and might conflict with other Util classes in the future.
Shall we change :identifier-util to :ballerina-identifiers
Utils -> IdentifierUtils ? WDUT?

Reply:
Then the package name of the class would be `io.ballerina.identifier.IdentifierUtils`. Here the `identifier` part seems to be repeating. Is that ok?
or shall we rename only the gradle project to ` :ballerina-identifiers`?